models/gif_cond_exp_multisynapse.h

```diff
@@ -130,14 +130,46 @@
   Synaptic parameters
     tau_syn   vector of double - Time constants of the synaptic conductance
                                  in ms.
-    E_ex       double - Excitatory reversal potential in mV.
-    E_in       double - Inhibitory reversal potential in mV.
+    E_rev      double vector - Reversal potential in mV.
 
   Integration parameters
     gsl_error_tol  double - This parameter controls the admissible error of the
                             GSL integrator. Reduce it if NEST complains about
                             numerical instabilities.
 
+  Example:
+
+  import nest
+  import numpy as np
+
+  neuron = nest.Create('gif_cond_exp_multisynapse')
+  nest.SetStatus(neuron, {'E_rev':[0.0, -85.0],
+                         'tau_syn':[4.0, 8.0]})
+
+  spike = nest.Create('spike_generator', params = {'spike_times':
+                                                 np.array([10.0])})
+
+  voltmeter = nest.Create('voltmeter', 1, {'withgid': True})
```

Comment:
I think I would leave the voltmeter out as well, it does not really tell us much about the model. You can also drop the plotting section towards the end of the example. Only up to the `Connect()` call is really interesting.